common-topics/vipps-landing-page.md

```diff
@@ -95,33 +99,52 @@ after completion of the payment. The "result page" is just the confirmation in
 Vipps. The `fallback` URL sent in the API request can therefore be the
 merchant's main URL, like `https://example.com`, etc.
 
-If the `skipLandingPage` property is set to `true`:
-* It will cause a push notification to be sent immediately to the given
-  phone number, without loading the landing page.
-* The user is not sent to a URL after completion of the payment.
-  The "result page" is just the confirmation in
-  Vipps.
-* If the sale unit is not whitelisted, the request will fail and an error
-  message will be returned.
-
-**Important:** When using `"skipLandingPage": true`:
-
-* The user is not able to provide a different phone number for completing the payment.
+**Important:** When using `"skipLandingPage": true` in the API request that
+initiates the payment:
+* Vipps will send a push notification immediately to the Vipps app
+  for the user with the specified phone number, without showing the landing page.
 * It is crucial to use the correct format for the user's phone number.
   If not, the payment will fail.
-* The user is not sent to a `fallback` URL after completion of the payment.
-  The "result page" is just the confirmation in Vipps.
+* The user is not able to provide a different phone number for completing the
+  payment. This means that a "Vipps for those under 15" (that can not pay
+  businesses) can not have someone else pay for them.
+* The user is not sent to a `fallback` URL (the result page) after completion
```

Comment:
Is that supposed to say "now"? Because, it seems like we are talking about what happens on the fallback page below

Reply:
Fixed it up bit, thanks.

Reply:
You have a typo below :-p  "Instead of the he "result page""
but it is more understandable

Reply:
I will fix it so we don't have to think about it